.git-blame-ignore-revs

```diff
@@ -0,0 +1,8 @@
+# To ignore these commits when using `git blame`, run the following command:
+# git config blame.ignoreRevsFile .git-blame-ignore-revs
+
+# Clippy and formatting fixes
+c582e23dfc79027362d8c4fa992493c10bb71caa
+
+# Remove UpdateContext's unused lifetime and reborrow method
+37714e8717730e21416a032441118262bf1f3c7b
```

Comment:
I believe these commits will change after merge and we have to commit them in a separate PR. One of them is already outdated